sites/all/modules/hr/hr_documents/hr_documents.module

```diff
@@ -95,3 +95,20 @@ function hr_documents_ctools_plugin_directory($module, $plugin) {
   }
 }
 
+/**
+ * Implements hook_field_widget_WIDGET_TYPE_form_alter().
+ */
+function hr_documents_field_widget_form_alter(&$element, &$form_state, $context) {
```

Comment:
Not really, it looks ugly
met vriendelijk groeten,
Peter Droogmans

On 25 Apr 2016 10:47, Guillaume Viguier-Just <notifications@github.com> wrote:In sites/all/modules/hr/hr_documents/hr_documents.module:
> @@ -95,3 +95,20 @@ function hr_documents_ctools_plugin_directory($module, $plugin) {
>    }
>  }
>  
> +/**
> + * Implements hook_field_widget_WIDGET_TYPE_form_alter().
> + */
> +function hr_documents_field_widget_form_alter(&$element, &$form_state, $context) {

Wouldn't it be enough to change the widget to a simple select (and would avoid custom code) ?

—You are receiving this because you were mentioned.Reply to this email directly or view it on GitHub